Core/src/org/sleuthkit/autopsy/geolocation/Bundle.properties-MERGED

```diff
@@ -3,8 +3,11 @@ CTL_GeolocationTopComponentAction=GeolocationTopComponent
 CTL_GeolocationTopComponent=Geolocation
 GLTopComponent_initilzation_error=An error occurred during waypoint initilization.  Geolocation data maybe incomplete.
 GLTopComponent_name=Geolocation
+MayWaypoint_ExternalViewer_label=Open in ExternalViewer
 OpenGeolocationAction_displayName=Geolocation
 OpenGeolocationAction_name=Geolocation
 RefreshPanel.refreshLabel.text=The geolocation data has been updated, the visualization may be out of date.
 RefreshPanel.refreshButton.text=Refresh View
 RefreshPanel.closeButton.text=
+MapPanel.cordLabel.text=jLabel1
```

Comment:
we do not need this as a Bundle message it the label can be switched to being plain text with an empty string, if we do leave it as a bundle property though it "jLabel1" should probably be replaced with the empty string.